govuk-one-login/authentication-tech-docs
# AP-134: Provide a plain english changelog to explain significant documentation changes to relying party developers

[AP-134](https://govukverify.atlassian.net/browse/AP-134)

## Why

It is hard to determine the change history of the documentation without exploring the git history in the repository, this is not easily accessible to our users who are typically developers wanting to follow the documentation to build integrations.

## What

Add a table of significant documentation updates

![image](https://github.com/govuk-one-login/authentication-tech-docs/assets/34273121/0483ac89-e4d6-4ab9-a017-228060b26275)


## Technical writer support

Yes

## How to review

- pull the branch down locally and run `./preview-with-docker.sh`
- open `http://localhost:345`
- review new content in context
- ensure links work and copy makes sense

## Confirm



[AP-134]: https://govukverify.atlassian.net/browse/AP-134?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
